automated-comments/elixir/general/parsing_error.md

```diff
@@ -0,0 +1,14 @@
+<!--
+requires 3 params:
+  line: line of the error
+  error: the error encountered
+  token: if a specific token is mentions by the parser
+-->
+
+Something went wrong while parsing the submission; most likely a "Syntax Error":
+
+Message: Parsing error on line: `%{line}`. Check near: `%{token}`
```

Comment:
The comment above seems to indicate that the `token` parameter could be empty: "if a specific token is mentions by the parser". Does that mean that this comment might be rendered as: "Check near: "?

Reply:
Ya, the token variable is not always set, is it possible to do something like `%{ defined?(token) ? ("Check near\'" + token + "\'") : "" }`? Or would it better to change it on the analyzer side?

Reply:
Good question, I'm not sure. @iHiD what would you prefer?

Reply:
The `%{}` won't execute code - it's purely string interpolation. So analyzer-side.

Reply:
Ok, will push changes in a little bit then

Reply:
See new commit, token is now appended to error in the analyzer.